### PR TITLE
Update hdb_sizes.json

### PR DESCRIPTION
**Pull request by KimForss**
I updated the hana sizing json to reflect this: https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/hana-vm-operations-storage#solutions-with-premium-storage-and-azure-write-accelerator-for-azure-m-series-virtual-machines

### CODE REVIEW

File: deploy/hdb_sizes.json [hunk 1]
```diff
@@ -1,393 +1,1052 @@
...
-                "mount_point" : "/usr/sap"
-            },
-            {
-                "name" : "backup",
-                "count" : 1,
-                "mount_point" : "/hana/logbackups"
-            }
-        ]
+{
+  "LargeInstance": {
```
Comment (rukawata) [inline deploy/hdb_sizes.json:2]: Please keep the 4 space as before :)

File: deploy/hdb_sizes.json [hunk 2]
```diff
@@ -1,393 +1,1052 @@
...
+        "count": 4,
+        "disk_type": "StandardSSD_LRS",
+        "size_gb": 1023,
+        "caching": "ReadOnly",
+        "write_accelerator": false,
+        "mount_point": "/hana/backup"
+      }
+    ]
+  },
+  "Standard_M32ts": {
```
Comment (rukawata) [inline deploy/hdb_sizes.json:392]: Do we need to keep `Standard_` prefix? Or keep it identical as in the Article and uses `M32ts`?

File: deploy/hdb_sizes.json (hunk 1 above)
Comment (KimForss) [reply deploy/hdb_sizes.json:2]: fixed - VSCode auto edits the json files for me

File: deploy/hdb_sizes.json (hunk 2 above)
Comment (KimForss) [reply deploy/hdb_sizes.json:392]: Like the article, I made the changes

**Comment by KimForss** [comment]
I updated the the sizing to reflect the latest

**Pull request merged**